general/g.tempfile/g.tempfile.html

```diff
@@ -10,10 +10,10 @@ <h2>DESCRIPTION</h2>
 
 <p>
 <em>g.tempfile</em>
-creates an unique file and prints the name. The user is required to provide
-a process-id which will be used as part of the name of the file.
-Most Unix shells provide a way to get the process id of the current shell.
-For /bin/sh and /bin/csh this is $$.
+creates an unique file or directory (<b>-f</b>) and prints the name.
+The user is required to provide a process-id which will be used as part of
+the name of the file or directory. Most Unix shells provide a way to get the
+process id of the current shell. For /bin/sh and /bin/csh this is $$.
```

Comment:
This is very shell/Bash-centric? Do you want to make it more universal or applicable to Python as part of this work?

Reply:
Agreed. See an attempt to update the manual in [fb30b7b](https://github.com/OSGeo/grass/pull/4397/commits/fb30b7be2e1633e815c93fa09f4a6179137702c5).